doc/source/conf.py

```diff
@@ -25,6 +25,16 @@
 sys.path.insert(0, str(Path("../..").resolve()))
 sys.path.insert(0, str(Path("../../src").resolve()))
 
+# Here provide the default values for the Python version range
+PYTHON_DEFAULT_MAX_VERSION = 3.13
```

Comment:
is there a reason this is called `PYTHON_DEFAULT_MAX_VERSION` and not just `PYTHON_MAX_VERSION`.  In other words, what is "default" about it?  I think this is set by the owner/maintainer, not a default, right?  Just a thought to increase understandability.